tests/Aspire.Playground.Tests/ProjectSpecificTests.cs

```diff
@@ -39,6 +39,10 @@ public async Task KafkaTest()
         await app.StartAsync();
         await app.WaitForResources().WaitAsync(TimeSpan.FromMinutes(2));
 
+        // Wait for the producer to start sending messages
+        await app.WaitForTextAsync("Hello, World! 10").WaitAsync(TimeSpan.FromMinutes(5));
```

Comment:
Where does the `10` come from? Do you just want to wait for `Hello, World!`?

Reply:
Hm yeah, that sounds better actually 👍